### PR TITLE
[PL] Complete polish translation of Plug lesson

### PR DESCRIPTION
**Pull request by Koziolek**

### CODE REVIEW

**Comment by doomspork** [comment]
Thanks @Koziolek!  @kdebowski could you take a peek at this?

File: pl/specifics/plug.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+I oto jest! Skonfigurowaliśmy nasz plug by obsługiwał żądania tak by dla ścieżki `/upload` wymagane były pola `"content"` i `"mimetype"`.  Tylko wtedy zostanie wykonany kod routera.
+
+Na chwilę obecną adres `/upload` nie jest specjalnie użyteczny, ale wiemy już jak stworzyć i włączyć nasz plug.
+
+## Uruchomienie aplikacji web
+
+Zanim uruchomimy naszą aplikację web musimy skonfigurować serwer, w tym przypadku będzie to Cowboy. Będzie to konfiguracja minimum która pozwoli nam na uruchomienie aplikacji, szczegółami zajmiemy się w kolejnych lekcjach.
+
+Rozpocznijmy od aktualizacji sekcji `application` w pliku `mix.exs` tak by wskazać Elixirowy naszą aplikację i ustawić jej zmienne środowiskowe. Po tych zmianach nasz plik powinien wyglądać mniej więcej tak:
```
Comment (kdebowski) [inline pl/specifics/plug.md]: Elixirowy - Elixirowi

**Comment by kdebowski** [comment]
@doomspork, @Koziolek I found only one typo.

**Comment by Koziolek** [comment]
Fixed

**Comment by Koziolek** [comment]
@doomspork this is ready to go :)

**Comment by doomspork** [comment]
Thank you @Koziolek & @kdebowski 👍

**Pull request merged**